pkg/ocm/ocm.go

```diff
@@ -3,16 +3,24 @@ package ocm
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"os"
 
 	_ "github.com/golang/mock/mockgen/model"
 	sdkcfg "github.com/openshift-online/ocm-cli/pkg/config"
 	sdk "github.com/openshift-online/ocm-sdk-go"
 
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	servicelog "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
 	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
 	hivev1 "github.com/openshift/hive/apis/hive/v1"
 )
 
+var (
+	sl_clusterHasGoneMissing = "https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/cluster_has_gone_missing.json"
```

Comment:
doesn't using `master` mean it might break in the future? isn't it better to have it pinned to a release / commit?

Reply:
yes thought of it as well. It makes definitely sense to evaluate it in the future. Pinning it to master will just take the latest template so far but makes it prone to failures. But I left it out for now

Reply:
@ramonbutter I think it will be better if we hard code the message instead of using the file from github. This will prevent issues if the messsages change or if github is down.

Reply:
You can use triple ticks to do that.
This would also mean that we will have to get mgmt approval for changing the SL here aswell

Reply:
I saw we put currently approved here and if it changes in the future we make a new PR with current wording.

Reply:
Agree.. having the hardcoded service log in JSON in there is the better option